apps/storefront/pnpm-lock.yaml

```diff
@@ -47,8 +47,8 @@ specifiers:
 dependencies:
   '@emotion/core': 10.1.1_react@17.0.1
   '@equinor/eds-core-react': 0.8.3_d56e21ef07fc37b6ee9b57ce562afcca
-  '@equinor/eds-icons': link:../../libraries/icons
```

Comment:
This file is dead code, so there should be no changes here, not sure how this got changed... We are in the process of removing this file altogether #1602.